firebase/firestore-py/constants.yaml

```diff
@@ -29,6 +29,9 @@ testers:
   gershmane@ramaz.org: 
     first: "Ezra"
     last: "Gershman"
+  jankelovitsb@ramaz.org:
+    first: "Barbara"
+    last: "Jankelovits"
```

Comment:
We... probably shouldn't commit this file. See if you can get rid of it using the BFG cleaner, which removes a file from _all commits_ in a repo. Then it'll disappear here too